### DISCUSSION TITLE
Unable to start my project

### DISCUSSION THREAD

**Discussion by akshithere** [honojs/hono#2480] [Q&A] [answered]
i am facing issue in the cloning template step:
![image](https://github.com/honojs/hono/assets/130065975/680954c6-eebe-4b86-a4af-b0b397f886a2)

**Comment by yusukebe** [comment, marked as answer]
Hi @akshithere 

Have you installed `git`?

**Reply by akshithere** [reply]
i believe yes

**Reply by akshithere** [reply]
apologies for the oversight i had not saved git in the path variable i just did that

**Reply by akshithere** [reply]
Its working now thanks a lot

**Reply by yusukebe** [reply]
Good!

**Reply by KaiBods** [reply]
Hi, @yusukebe, I am getting the exact same error although trying to use the nodejs template. 
However, I don't have git installed, is this a prerequisite?

**Reply by yusukebe** [reply]
@KaiBods Yes. You need ‘git’.

**Reply by KaiBods** [reply]
Apologies @yusukebe , I had a brain-fart! I was trying to get things running manually and wasn't paying attention and trying to create when I didn't need to. The reason it wasn't working for me was because we are using Ampt (who HEAVILY recommended Hono to us), but it needs their SDK to work which I was completely overlooking. Apologies for wasting your time.
This is our first steps into using Hono, and am really excited to start delving deeper :)
